mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -20,8 +32,27 @@ AUI.add(
 					initializer: function(config) {
 						var instance = this;
 
+						new A.CharCounter(
+							{
+								counter: '.counter',
+								input: '.input-item',
+								maxLength: 35
+							});
+
 						var todoList = instance.one('.task-container ul');
 
+						var finishedTask = A.one('.finished-task');
+
+						var remainingDiv = instance.one('.remaining-tasks-count');
```

Comment:
Instead of ```instance.one()```, use ```instance.byId()```, ```A.one()``` or ```node.one()```, as previously requested in https://github.com/markocikos/liferay-internship/pull/133#discussion_r118944623. There are 10+ usages if in this PR. You even stated you are doing this in commit name!